plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -46,12 +46,10 @@ import {
 } from '@opentelemetry/semantic-conventions';
 import { VERSION } from './version';
 
-const PG_POOL_COMPONENT = 'pg-pool';
```

Comment:
I think we should consider using an attribute to keep track of the `pg-pool` vs `pg` component instead of having it at the name level

Reply:
If we remove PG_POOL_COMPONENT, we'll lose this information. We can't know if the user called pg.connect() of pg-pool.connect(). I would like to see what other @open-telemetry/javascript-approvers think.

Reply:
I think the correct way to reflect this would be the instrumentation scope. So 1 tracer for pg and one for pg-pool.
But not sure if it is important at all to have this info available.

Reply:
I would love to see it captured as a scope attribute.

@xoscar if you want to include it in this one, please go ahead and add it.

Since currently it is only being published as part of the span name which isn't very useful anyway, I think maybe we can postpone the introduction of a new attribute to a follow-up PR. We can open an issue for that and merge the current PR.

@haddasbronfman @Flarna WDYT?